web/client/actions/queryform.js

```diff
@@ -255,9 +255,10 @@ function query(seachURL, data) {
     };*/
 }
 
-function reset() {
+function reset(skip) {
     return {
-        type: QUERY_FORM_RESET
+        type: QUERY_FORM_RESET,
+        skip
```

Comment:
I can't see skip anywhere else

Reply:
It's used in query reducer

Reply:
please add a test that avoid any change on this. Explain well why it's necessary in the test.